src/client-python/reportclient/debuginfo.py

```diff
@@ -420,6 +418,10 @@ def download(self, files, download_exact_files=False):
 
             progress_observer.downloaded_pkgs += 1
 
+        for root, dirs, files in os.walk(self.cachedir):
```

Comment:
Let’s not bloat the function any more and move this out to the caller script.

Reply:
You mean to src/plugins/abrt-action-install-debuginfo.in in abrt package? That would break encapsulation.

Reply:
Yes, that’s what I mean, since it passes down the cache directory.

Reply:
I assume that’s what “breaking encapsulation” means.